Lessons/Lesson 3/Solutions/Lesson_3_Solution_JBH.cs

```diff
@@ -0,0 +1,130 @@
+﻿using System;
+using System.Diagnostics.CodeAnalysis;
+using System.Runtime.InteropServices;
+
+double num1 = 0, num2 = 0;  // gloabl variables for basic calculator portion.
+int index = 0, userChoice; // global variables for fibbonacci function.
+
+void PrintResult(double varToPrint)
+{
+    Console.Clear();
+    Console.WriteLine("Your result is: " + varToPrint);
+}
+
+void PrintMenu()
+{
+    Console.Write("Please select an operation that you would like to perform from the menu:\n\n" +
```

Comment:
format you long console.Write
Console.Write("Line 1\n"+
                        "line2\n" +
                        "ect..."